src/main/kotlin/no/nav/syfo/behandlendeenhet/api/TildelHistorikkResponseDTO.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.syfo.behandlendeenhet.api
+
+import no.nav.syfo.behandlendeenhet.EnhetService.Companion.SYSTEM_USER_IDENT
+import no.nav.syfo.behandlendeenhet.api.TildelOppfolgingsenhetHistorikkType.*
+import no.nav.syfo.behandlendeenhet.domain.Oppfolgingsenhet
+import java.time.LocalDateTime
+
+data class TildelHistorikkResponseDTO(
+    val oppfolgingsenheter: List<TildelOppfolgingsenhetHistorikkDTO>,
+)
+
+enum class TildelOppfolgingsenhetHistorikkType {
+    TILDELT_ANNEN_ENHET_AV_VEILEDER,
+    TILDELT_TILBAKE_TIL_ENHET_AV_VEILEDER,
+    TILDELT_TILBAKE_TIL_ENHET_AV_SYSTEM,
+}
+
+abstract class TildelOppfolgingsenhetHistorikkDTO {

```

Comment:
Kanskje vi ikke trenger `Tildel…` i navnet? 😊 
```suggestion
abstract class OppfolgingsenhetHistorikkDTO {

```

Reply:
Det er nå endret til `TildeltOppfolgingsenhetHistorikkDTO` og så er `tildelt` det som blir brukt andre steder slik at det er nogenlunde konsekvent (i stedet for f.eks. `tildelingshistorikk` e.l.)